### PR TITLE
JSDK-3021 Document Large Group Rooms behavior.

### PR DESCRIPTION
**Pull request by manjeshbhargav**
@innerverse @idelgado @ceaglest 

This PR documents the behavior of Large Group Rooms. I will open a separate PR for integration tests.

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Review by innerverse — APPROVED** [review]

File: lib/util/twilio-video-errors.js [hunk 1]
```diff
@@ -587,7 +587,7 @@ Object.defineProperty(TwilioErrorByCode, 53202, { value: ParticipantIdentityChar
  */
 class ParticipantMaxTracksExceededError extends TwilioError {
   constructor() {
-    super(53203, 'Participant has too many Tracks');
+    super(53203, 'Published Tracks at maximum capacity');
```
Comment (idelgado) [inline lib/util/twilio-video-errors.js:590]: Can you mention this error somewhere in the blueprint. Probably best under Functional Requirements: https://docs.google.com/document/d/1kaHlFTD4XfdpqcqA5QgMbX_EF83I0eANw-6wKF7i1_4/edit

File: lib/localparticipant.js [hunk 2]
```diff
@@ -362,7 +364,9 @@ class LocalParticipant extends Participant {
    * @param {MediaStreamTrackPublishOptions} [options] - The options for publishing
    *   the MediaStreamTrack
    * @returns {Promise<LocalTrackPublication>} - Resolves with the corresponding
-   *   {@link LocalTrackPublication} if successful
+   *   {@link LocalTrackPublication} if successful; In a Large Group Room (Maximum
```
Comment (idelgado) [inline lib/localparticipant.js:367]: Maybe worth clarifying the behavior when a publication request is made that exceeds 16, for example, you have 15 tracks, and someone attempts to publish 2 more once this comment is resolved: https://docs.google.com/document/d/1kaHlFTD4XfdpqcqA5QgMbX_EF83I0eANw-6wKF7i1_4/edit?disco=AAAAKS-lILo

**Review by idelgado — CHANGES_REQUESTED** [review]

File: lib/util/twilio-video-errors.js (hunk 1 above)
Comment (idelgado) [reply lib/util/twilio-video-errors.js:590]: Is there a corresponding error code PR that adds this as a cause?

Comment (manjeshbhargav) [reply lib/util/twilio-video-errors.js:590]: @idelgado It's already added in the blueprint [here](https://docs.google.com/document/d/1kaHlFTD4XfdpqcqA5QgMbX_EF83I0eANw-6wKF7i1_4/edit#heading=h.a4zxg4wb8s92). I will open an error code pr.

**Review by idelgado — APPROVED** [review]

**Pull request merged**